sheetScript.gs

```diff
@@ -200,37 +200,40 @@ function updatePlaylists(sheet) {
   }
 }
 
-function addVideosToPlaylist(playlistId, videoIds) {
-  var errorCount = 0;
+function addVideosToPlaylist(playlistId, videoIds, idx = 0, successCount = 0, errorCount = 0) {
   var totalVids = videoIds.length;
-  if (videoIds.length < maxVideos) {
-    for (var i = 0; i < totalVids; i++) {
-      try {
-        YouTube.PlaylistItems.insert({
-          snippet: {
-            playlistId: playlistId,
-            resourceId: {
-              videoId: videoIds[i],
-              kind: 'youtube#video'
-            }
+  var success = 1;
+  if (0 < totalVids && totalVids < maxVideos) {
```

Comment:
With just an `if...else`, the script returns the >200 videos error if we aren't adding any videos to the playlist. This should be a separate `else if` condition, which just returns, maybe saying something like `No videos to add`.

Reply:
I had already updated this but forgot to push. Sorry about that.